movies/src/helpers/sortBy.js

```diff
@@ -0,0 +1,28 @@
+const sortBy = (sortType) => (a, b) => {
```

Comment:
why are you using client sorting? seems like endpoint supports server side sorting